### ISSUE TITLE
Desktop default "View Mode"

### ISSUE THREAD

**Issue by C4rb0n** [Ylianst/MeshCentral#4980]
It would be nice if there was a way to configure the default view mode (global and per-device override) so that the initial user experience can be enhanced -- not only for less tech-savvy users, but also so savvy users who don't want to have to toggle the view mode on every connection.  User settings should further be able to override the settings.  Specifically, being able to set the default view mode to either:

1. Actual size/resolution (of the remote display) -- to achieve best clarity
2. Local viewport size
3. Maximized
4. Full Screen

Quality and scaling defaults could also be included.
 
It would also be nice if there was an option to auto-hide the top and bottom bars when size of the local viewport would result in them overlapping the remote screen.

Thanks

**Comment by C4rb0n** [comment]
When you click on a device and go to the "Desktop" tab, it's the south-east pointing arrow on the right.  If you hover your mouse pointer over it, the tooltip says "Toggle View Mode."

![image](https://user-images.githubusercontent.com/23385632/218622508-a9f0bbd0-1d76-4bba-af62-7d597defdfee.png)

**Comment by compumatter** [comment]
I agree.  This default of 40% quality has clients immediately saying its horrible.  Great for admins as they are willing to deal with it but when you allow your clients to use it as a solution for their own network they immediately balk at the resolution. 

So yes, there is a need to set default quality / resolution to improve it to the degree possible on first login and not require the end user move to manual tweaks right up front.

**Comment by VGkav** [comment]
It would save me thousands of clicks if I could somehow set the connection to maximized when connecting to a client :D

**Comment by redddcyclone** [comment]
I agree with this one
